client/src/modules/schemes/modules/relation-editor/RelationEditor.vue

```diff
@@ -0,0 +1,295 @@
+<template>
+    <div class="content">
+        <!-- The dialog for deleting a relation -->
+        <ConfirmDialog
+            @confirm="deleteRelation"
+            @cancel="deleteRelationDialog = false"
+            :show="deleteRelationDialog"
+            :title="$t('schemes.relationEditor.deleteDialog.title')"
+            :description="$t('schemes.relationEditor.deleteDialog.description')"
+        ></ConfirmDialog>
+
+        <div class="underlined-title">
+            {{ $t("schemes.relationEditor.title") }}
+        </div>
+        <ScrollPanel class="scroll-panel">
+            <!-- The main attributes -->
+            <label class="main-attribute">
+                <span>{{ $t("schemes.relationEditor.name") }}</span>
+                <input v-if="createMode" id="relation-name-input" v-model="modifiedRelation.name" />
+                <span v-else> {{ modifiedRelation.name }}</span>
+            </label>
+
+            <!-- The optional parameters -->
+            <div class="attributes-action-bar">
+                <h4>{{ $t("schemes.relationEditor.attributes") }}</h4>
+                <svg class="add-attribute" @click="addAttribute">
+                    <use :xlink:href="`${require('@/assets/img/icons.svg')}#plus-bold`"></use>
+                </svg>
+            </div>
+
+            <AttributeView
+                v-for="(attribute, index) in modifiedRelation.attributes"
+                :key="`${relation.name}-${objectUUID(attribute)}`"
+                v-model:name="attribute.name"
+                v-model:mandatory="attribute.mandatory"
+                v-model:defaultValue="attribute.defaultValue"
+                v-model:datatype="attribute.datatype"
+                @delete="deleteAttribute(index)"
+            />
+
+            <!-- The optional parameters -->
+            <div class="attributes-action-bar">
+                <h4>{{ $t("schemes.relationEditor.connections") }}</h4>
+                <svg class="add-attribute" @click="addConnection">
+                    <use :xlink:href="`${require('@/assets/img/icons.svg')}#plus-bold`"></use>
+                </svg>
+            </div>
+
+            <ConnectionView
+                v-for="(connection, index) in modifiedRelation.connections"
+                :key="`${connection.name}-${objectUUID(connection)}`"
+                v-model:from="connection.from"
+                v-model:to="connection.to"
+                @delete="deleteConnection(index)"
+            />
+
+            <!-- The save button --->
```

Comment:
The save-button will not be enabled if you just change the direction of an existing connection